site/content/en/docs/Examples/custom-controller.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Custom Controller for Agones Game Servers"
+linkTitle: "Custom Controller"
+date:
+publishDate:
+description: >
+  This example shows how to deploy and run the Custom Controller example on Agones, a tool to monitor the running dedicated game servers on Kubernetes.
+---
+
+## Deploy the Custom Controller
+
+To get started, update the [deployment.yaml](https://github.com/googleforgames/agones/blob/main/examples/custom-controller/deployment.yaml) file to use your Docker image, then apply it to your cluster:
```

Comment:
I've made the changes as per your suggestion

Reply:
If we include the information about the custom deployment, there is a possibility that it could impact the production environment due to build and push details?